ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/impl/ClientImplTest.java

```diff
@@ -87,7 +87,7 @@ public void shouldSetUserAgentAndAcceptHeaders() {
     client.streamQuery("SELECT * from STREAM1 EMIT CHANGES;");
     assertThat(headers.size(), is(2));
     assertThat(headers.containsKey(USER_AGENT.toString()), is(true));
-    assertThat(headers.get(USER_AGENT.toString()).matches("ksqlDB Java Client v\\d\\.\\d\\.\\d.*"),
+    assertThat(headers.get(USER_AGENT.toString()).matches("ksqlDB Java Client v\\d*\\.\\d*\\.\\d*.*"),
```

Comment:
@swist Thanks for looking into this -- just realizing that this should be `+` not `*` ?

Reply:
d'oh, I even did the interview yesterday. I'll fix it up